torchx/README.md

```diff
@@ -27,13 +27,22 @@ end
 ```
 
 We will automatically download a precompiled version of `LibTorch` that runs on the CPU.
-If you want to use another version, [download the .zip file of another `LibTorch` version
-from the official website](https://pytorch.org/get-started/locally/), unpack it to a
-directory of your choice and set the `LIBTORCH_DIR` [environment variable](https://en.wikipedia.org/wiki/Environment_variable)
-to point to the unpacked directory.
-
-Once downloaded, we will compile `Torchx` bindings. You will need `make`, `cmake` (3.12+)
-and a `C++` compiler. If running on Windows, you will need:
+If you want to use another version, please use LIBTORCH_VER to choose the different versions. The current
```

Comment:
```suggestion
If you want to use another version, please use `LIBTORCH_VERSION` to choose the different versions. The current
```

Can we please use `LIBTORCH_VERSION` as the variable name? No abbreviations:

Reply:
sure